src/process_utility.c

```diff
@@ -328,19 +326,26 @@ process_drop_table(DropStmt *stmt)
 			ht = hypertable_cache_get_entry(hcache, relid);
 			if (NULL != ht)
 			{
+				CatalogSecurityContext sec_ctx;
```

Comment:
could we just move this to the top level of this function? I don't see a problem with doing that right now, and it would avoid a lot of repetitive stuff.

Reply:
done